apisix/core/config_etcd.lua

```diff
@@ -172,8 +171,8 @@ local function sync_data(self)
             self.values_hash = nil
         end
 
-        self.values = new_tab(#dir_res.nodes, 0)
-        self.values_hash = new_tab(0, #dir_res.nodes)
+        self.values = table.new(#dir_res.nodes, 0)
+        self.values_hash = table.new(0, #dir_res.nodes)
```

Comment:
These changes have nothing to do with this PR?

Reply:
reverted